e2e/claims.service.e2e.ts

```diff
@@ -913,6 +984,58 @@ describe('Сlaim tests', () => {
         expect(result.errors).toHaveLength(0);
         expect(result.isVerified).toBe(true);
       });
+      test('verifyVc should return a credential expired error if credential is expired', async () => {
+        await signerService.connect(rootOwner, ProviderType.PrivateKey);
+        const issuerFields = [];
+        const vc = await createExampleSignedCredential(
+          issuerFields,
+          `${vcExpired}.${root}`,
+          new Date(Date.now() + 10000)
+        );
+        nock(vc.credentialStatus?.statusListCredential as string)
+          .get('')
+          .reply(200, undefined);
+        const delay = (ms) => new Promise((res) => setTimeout(res, ms));
+        await delay(11000);
```

Comment:
A minor point, but does the delay need to be this long? 11sec is quite a bit of time for a test. Would 1 or 2 seconds (or even less) do?

Reply:
I'll try decreasing the numbers. The expiration date is set to be 11 seconds from the time of issuance to allow time for issuance etc,  so I just want to make sure that we are doing the check after that time passes. Will try to decrease.